correios/client.py

```diff
@@ -61,22 +60,11 @@
     ),
 }
 
-CorreiosAPIError = namedtuple('CorreiosAPIError', ['regex', 'exception'])
-
-ERRORS = (
-    CorreiosAPIError(
-        regex=re.compile(r"autenticacao"),
-        exception=AuthenticationError,
-    ),
-    CorreiosAPIError(
-        regex=re.compile(r"^O Cartão de Postagem.*Cancelado.$"),
-        exception=CanceledPostingCardError,
-    ),
-    CorreiosAPIError(
-        regex=re.compile(r"^Cartao de Postagem inexistente"),
-        exception=NonexistentPostingCardError,
-    ),
-)
+ERRORS = {
+    re.compile(r"autenticacao"): AuthenticationError,
+    re.compile(r"^O Cartão de Postagem.*Cancelado.$"): CanceledPostingCardError,
+    re.compile(r"^Cartao de Postagem inexistente"): NonexistentPostingCardError,
+}
```

Comment:
Mutch better, hã? 🙂

Reply:
Yep, without the error message this is the best choice :)